sql/core/src/test/scala/org/apache/spark/sql/execution/command/MsckRepairTableParserSuite.scala

```diff
@@ -31,6 +31,15 @@ class MsckRepairTableParserSuite extends AnalysisTest {
         enableDropPartitions = false))
   }
 
+  test("repair a table without MSCK keyword") {
+    comparePlans(
+      parsePlan("REPAIR TABLE a.b.c"),
+      RepairTable(
+        UnresolvedTable(Seq("a", "b", "c"), "MSCK REPAIR TABLE", None),
```

Comment:
Shouldn't we change the hint if `REPAIR TABLE` becomes the main way of invoking the command?

Reply:
Agreed, this should be updated. Will do this now and then push again.

Reply:
```suggestion
        UnresolvedTable(Seq("a", "b", "c"), "REPAIR TABLE", None),
```

Reply:
According to the agreement here, I'll merge @MaxGekk 's suggestion and backport to branch-3.4 for Apache Spark 3.4 release.